pkgs/development/libraries/libewf-legacy/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, fetchurl
+, fetchpatch
+, stdenv
+, zlib
+, openssl
+, libuuid
+, pkg-config
+, bzip2
+}:
+
+stdenv.mkDerivation rec {
+  version = "20140814";
+  pname = "libewf-ewf";
+
+  src = fetchurl {
+    url = "https://github.com/libyal/libewf-legacy/releases/download/${version}/libewf-${version}.tar.gz";
+    hash = "sha256-OM3QXwnaIDeo66UNjzmu6to53SxgCMn/rE9VTPlX5BQ=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ zlib openssl libuuid ]
+    ++ lib.optionals stdenv.isDarwin [ bzip2 ];
```

Comment:
Why?

Reply:
I followed the same "style" of [libewf](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/libewf/default.nix) file. libewf-legacy is an old version of libewf but it is needed since a lot of applications work with old versions of libewf. For this reason, libewf developers decided to create this "legacy" fork that is still maintained.

Reply:
It is a bit strange, tbh. 
Why the developers would use bzip2 only for Darwin?

Reply:
We should ask to @7c6f434c

Reply:
https://github.com/NixOS/nixpkgs/pull/145552 — this is below my level of caring for Darwin, it fixes the build on a platform I avoid, not too horrible, good enough for me. I guess enabling this unconditionally wouldn't break things anyway.